qa/logging-config/src/test/java/org/opensearch/common/logging/OpenSearchJsonLayoutTests.java

```diff
@@ -61,7 +61,7 @@ public void testLayout() {
                 "\"component\": \"%c{1.}\", " +
                 "\"cluster.name\": \"${sys:opensearch.logs.cluster_name}\", " +
                 "\"node.name\": \"%node_name\", " +
-                "\"message\": \"%notEmpty{%enc{%marker}{JSON} }%enc{%.-10000m}{JSON}\"" +
+                "\"message\": \"%notEmpty{%enc{%marker}{JSON} }%enc{%.-100000m}{JSON}\"" +
```

Comment:
@djboris9 the 10k default is sufficiently large for 99% of cases, I would suggest to keep the default unchanged but introduce the attributed (fe. `messagelength` or alike) to the `OpenSearchJsonLayout` so those who need it could adjust the limit.

Reply:
Thanks for your comment @reta , I would have expected to get this truncating issue much more often, as the `audit_trace_resolved_indices` can be large and in my case only 250 affected indices are enough to break it.
I will try to make this configurable.